db/migrate/20221219171622_create_parcels.rb

```diff
@@ -0,0 +1,18 @@
+class CreateParcels < ActiveRecord::Migration[6.0]
+  def change
+    create_table :parcels do |t|
+      t.string :geoid20,null:false
+      t.string :zip 
+      t.integer :tract ,null:false
+      t.string :block ,null:false
+      t.string :lat, null:true
+      t.string :long , null:true
```

Comment:
spaces should be after commas
not sure why we don't have rubocop on migrations. Maybe try turning on?
If that doesn't work then at least get a code beautifier in your IDE to catch this stuff?

Reply:
ok will  look into getting a code linter.